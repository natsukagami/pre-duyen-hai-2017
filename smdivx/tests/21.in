38 170875021823022
