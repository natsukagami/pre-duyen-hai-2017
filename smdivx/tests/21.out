10000000000000000000000168043260561122
